app.js

```diff
@@ -0,0 +1,4 @@
+const createImage = function () {
+    let url = 'https://source.unsplash.com/random/' + width +'/'+ height;
+    let newImage = document.createElement('img') ;
+    newImage.setAttribute('src', url);

```

Comment:
I think you missed some lines of your code.
think of adding a click event to the button, so when someone click the button it will create the image.